server/apps/templates/content_templates.py

```diff
@@ -78,6 +78,7 @@ class ContentTemplatesResource(Resource):
         }},
         'last_run': {'type': 'datetime', 'readonly': True},
         'next_run': {'type': 'datetime', 'readonly': True},
+        'headline_limit': {'type': 'integer', 'default': 0},
```

Comment:
imo it would be better to have there `limits` as a dictionary so that we don't have to update schema whenever we add a new one. eg. for `slugline` that should be there